src/pages/Login/login.jsx

```diff
@@ -1,10 +1,10 @@
 import React, { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import { useLoginUserMutation } from '../services/authApi';
-import { ROUTES } from '../constants/routerPaths';
-import LoginForm from '../components/Forms/LoginForm';
-import SharedTypography from '../components/shared/Text/SharedText';
-import { AUTH_ERRORS } from '../constants/auth.constants';
+import { useLoginUserMutation } from '../../services/authApi.js';
```

Comment:
organize your imports - Grouping same kind of imports (for example 3rd party in the beginning of the file separate with an empty line and then constants for example).
the are libraries that do this